sdk/communication/communication-chat/package.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "@azure/communication-chat",
-  "version": "1.0.0-beta.5",
+  "version": "1.0.0-beta.6",
```

Comment:
Same question here, can we stick with beta.5?

Reply:
@ankitarorabit @RezaJooyandeh what's the rationale behind this exactly?
@DominikMe i'm also going to update everything else to go to 6 now

Reply:
@RezaJooyandeh We've decided to stop trying a versioning lock-step, haven't we?